### PR TITLE
PAB-4353: Update doc for new Paste command

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

**Review by nabu-nnat — APPROVED** [review]

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md [hunk 1]
```diff
@@ -234,17 +234,31 @@ On the **Models** tab of the model manager, click the actions menu of the model
...
+### Pasting a model {#pasting-a-model}
+
+Instead of uploading a model, you can also paste the JSON code for a model from the clipboard.
+
+#### To paste a model {#to-paste-a-model}
+
+On the **Models** tab of the model manager, click **Import model** in the toolbar, and then click **Paste**.
+
+{{< c8y-admon-info>}}
+The **Paste** command is enabled only if the clipboard contains content, and only if your browser allows clipboard access. When you click **Import model** for the first time, some browsers ask if you want to allow or block clipboard access. In other browsers, such as Firefox, you must manually allow it in the browser settings.
```
[the remark this answers is not in the document]
Comment (Rob-J-SAG) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:258]: The Cumulocity doc references pasting from the clipboard in multiple other places and never explains the quirks of different browsers interacting with the clipboard. We should not do so here either.

[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:258]: @nabu-nnat  and @MWindel , what do you think? Do you agree to remove this? That is, only keep this: "The **Paste** command is enabled only if the clipboard contains content" ?

[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:258]: > I agree, we do not need this. Just keeping the first sentence should be sufficient.

So you also want to keep this?
"and only if your browser allows clipboard access. "

[the remark this answers is not in the document]
Comment (Rob-J-SAG) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:258]: I think we just want "The **Paste** command is enabled only if the clipboard contains content."

[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md]: ok, removed.

**Pull request merged**